docs/spec_draft.md

```diff
@@ -582,6 +583,39 @@ Produces a `result` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values from one element type to another
+on `operand` tensor.
```

Comment:
As discussed offline, this needs to go into details of what it means to convert from one element type to another.